From f3f7fbd95f9b0f96e7f552091ee41cccbaf739ca Mon Sep 17 00:00:00 2001
From: Alexandru Costache <alexandru@balena.io>
Date: Fri, 11 Jul 2025 08:31:20 +0000
Subject: [PATCH] Makefile: update path for usrmerge, to fix build

Upstream-status: Inappropriate [configuration]
Signed-off-by: Alexandru Costache <alexandru@balena.io>
---
 Makefile | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/Makefile b/Makefile
index 5c2f11f..fb6a5dd 100644
--- a/Makefile
+++ b/Makefile
@@ -74,12 +74,12 @@ checkpatch-staging:
 	@./scripts/checkpatch.sh --cached
 
 install:
-	$(echo) '  INSTALL ${DESTDIR}/lib/optee_armtz'
-	$(q)mkdir -p ${DESTDIR}/lib/optee_armtz
-	$(q)find $(out-dir) -name \*.ta -exec cp {} ${DESTDIR}/lib/optee_armtz \;
-	$(echo) '  INSTALL ${DESTDIR}/bin'
-	$(q)mkdir -p ${DESTDIR}/bin
-	$(q)cp $(out-dir)/xtest/xtest ${DESTDIR}/bin
+	$(echo) '  INSTALL ${DESTDIR}/usr/lib/optee_armtz'
+	$(q)mkdir -p ${DESTDIR}/usr/lib/optee_armtz
+	$(q)find $(out-dir) -name \*.ta -exec cp {} ${DESTDIR}/usr/lib/optee_armtz \;
+	$(echo) '  INSTALL ${DESTDIR}/usr/bin'
+	$(q)mkdir -p ${DESTDIR}/usr/bin
+	$(q)cp $(out-dir)/xtest/xtest ${DESTDIR}/usr/bin
 	$(echo) '  INSTALL ${DESTDIR}/$(CFG_TEE_PLUGIN_LOAD_PATH)'
 	$(q)mkdir -p ${DESTDIR}/$(CFG_TEE_PLUGIN_LOAD_PATH)
 	$(q)cp $(out-dir)/supp_plugin/*.plugin ${DESTDIR}/$(CFG_TEE_PLUGIN_LOAD_PATH)
-- 
2.34.1

